test/integration/functional_test.go

```diff
@@ -816,8 +836,34 @@ func validateSSHCmd(ctx context.Context, t *testing.T, profile string) {
 	if NoneDriver() {
 		t.Skipf("skipping: ssh unsupported by none")
 	}
-	want := "hello\n"
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", fmt.Sprintf("echo hello")))
+	want := "/home/docker\n"
```

Comment:
I don't understand the point of this new test. Why is /home/docker a critical testing value? The echo command seemed a better indicator of badness, to be honest.